src/test/groovy/com/ofg/geb/AcceptanceSwaggerUISpec.groovy

```diff
@@ -0,0 +1,54 @@
+package com.ofg.geb
```

Comment:
I would move it to `com.ofg.twitter.geb` package. For now it is specific to that application. Maybe in the future we would like to move common things to acceptance-test submodule in micro-infra-spring.